### ISSUE TITLE
3.6.1 Special characters wrongly displayed in runner UI assertions

### ISSUE THREAD

**Issue by gustawx** [cypress-io/cypress#5679]
### Current behavior:

![cypress](https://user-images.githubusercontent.com/13650820/68743031-d52d6280-05f1-11ea-9541-ed2608b1cc4e.PNG)

### Desired behavior:

Special characters properly displayed

### Steps to reproduce: (app code and test code)

for e.g. use assertion with regex:
should('match', /width: (50.*|49.*|48.*)%;/)

### Versions

3.6.1

**Comment by jennifer-shehane** [comment]
Verified in 3.8.0.

```js
it("prints", () => {
  cy.wrap('foo').should('match', /width: (50.*|49.*|48.*)%;/)
})
```

<img width="398" alt="Screen Shot 2019-12-17 at 2 26 29 PM" src="https://user-images.githubusercontent.com/1271364/70975926-4b355580-20d9-11ea-8318-3bb74a16fbc1.png">

This is due to our Command logging supporting markdown as explained here: https://github.com/cypress-io/cypress/issues/2134#issuecomment-405263976 So is an unfortunate side effect.

**Comment by gerardjorgensen** [comment]
Hi, can I work on this?

**Comment by jennifer-shehane** [comment]
@gerardjorgensen Yes we would love an open PR - even one that is a work in progress is fine. Check out our [contributing doc](https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md) and ask us if you get stuck.

**Comment by gerardjorgensen** [comment]
I'm unfortunately stuck figuring out where the special character is converted Markdown. I looked at https://github.com/cypress-io/cypress/blob/develop/packages/reporter/src/commands/command.tsx#L106 but from there I'm not sure where to go.

**Comment by jennifer-shehane** [comment]
It's done here, using `markdown-it` https://github.com/cypress-io/cypress/blob/cli-readme-md-typo/packages/reporter/src/commands/command.tsx#L23:L23

**Comment by jennifer-shehane** [comment]
This is still an issue in 9.5.3.
